certified-connectors/My Acclaro/README.md

```diff
@@ -0,0 +1,81 @@
+# My Acclaro
```

Comment:
Readme is not in the correct format. Please refer to this link to see a template of the readme as well as what are the required sections and their limitations. If you have any questions please feel free to contact us.

Reply:
this was already approved in my previous PR. I just wanted to add the `intro.md` file into this PR, which was requested as part of the ISV Studio certification (different phase of the certification process)